SpaceDuck.UserService/SpaceDuck.UserService/Migrations/ApplicationIdentity/20200329142657_Initial.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace SpaceDuck.UserService.Migrations.ApplicationIdentity
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "AspNetRoles",
```

Comment:
Do wszystkich nazw tabel i PK dodajemy "AspNet". Jest to konieczne?

Reply:
to jest  specyfika korzystania z biblioteki Microsoft.AspNetCore.Identity